tests/acceptance/blocks/filters.js

```diff
@@ -0,0 +1,36 @@
+import { Selector, t } from 'testcafe';
+
+/**
+ * Models the user interaction with filters
+ */
+class Filters {
```

Comment:
Would it make sense to call this `CollapsibleFilters`? The class seems specific to that use-case. We might want to create a `Filters` class later on for the static, non-collapsible filters.

Reply:
ya, that makes more sense